app/controllers/orders/view.js

```diff
@@ -1,9 +1,26 @@
 import Controller from '@ember/controller';
 
 export default Controller.extend({
+  isLoading: false,
+
   actions: {
     downloadInvoice() {
       window.print();
+    },
+    downloadTickets() {
+      this.set('isLoading', true);
+      this.get('loader')
+        .load(`/tickets/${this.get('model.order.identifier')}`, { withoutPrefix: true })
```

Comment:
`{ withoutPrefix: true }` 
Why?
Also, have you tested this locally?

Reply:
Yup. It works locally. I've actually routed the tickets to `/tickets/<order_identifier>`. Do you want me to route it to `v1/tickets/<order_identifier>`?

Reply:
Wait did you mean the ticket routing or `isLoading`?

Reply:
Wait, did you mean `isLoading` or the ticket routing? I actually have a problem loading the PDF in the browser. For some reason, it loads for a very long time and it throws an exception. But half of the times, the PDF is loaded. Can you check this?

Reply:
@CosmicCoder96